Doc/c-api/object.rst

```diff
@@ -179,6 +179,13 @@ Object Protocol
    If *o1* and *o2* are the same object, :c:func:`PyObject_RichCompareBool`
    will always return ``1`` for :const:`Py_EQ` and ``0`` for :const:`Py_NE`.
 
+.. c:function:: PyObject* PyObject_Format(PyObject *obj, PyObject *format_spec)
```

Comment:
Weird that we're inconsistent in putting the space before or after the `*`, but the next two functions do the same thing.

Reply:
The relevant style guide would be PEP 7, but it says nothing about this issue. All the examples use `PyObject *x`, though.